### ISSUE TITLE
Maintenance mode after update to Polls 7.1.0

### ISSUE THREAD

**Issue by dartcafe** [nextcloud/polls#3551]
### Announcement

Something is wrong with the NC updater I suspect, so this error leaves the system in maintenance mode. 

### You may find a log entry which says: 
```
TypeError: OCA\Polls\Db\UserMapper::__construct():
  Argument #2 ($userManager) must be of type OCP\IUserManager, OC\Session\Memory given, 
  called in lib/AppInfo/Application.php on line 158 and defined in lib/Db/UserMapper.php:60
```

Solution: run `occ upgrade` and disable the maintanance mode with `occ maintenance:moe --off`

See #3547 and #3526

**Comment by dartcafe** [comment]
For me it looks like a bug in the NC installer. If I am wrong, I apprechiate any hint for avoiding that.

**Comment by septatrix** [comment]
Is it possible to unpublish/remove this version before more people encounter this bug?

**Comment by XueSheng-GIT** [comment]
@dartcafe Just updated to polls 7.1.1 on NC 29.0.2 (did previously not install 7.1.0) and hitting this issue (system being in maintenance mode).

Seems this issue is not fixed with 7.1.1.

**Comment by dartcafe** [comment]
Thanks fror reporting, but please oeben another issue and provide some information from the logs.
